lib/MusicBrainz/Server/Edit/Annotation/Edit.pm

```diff
@@ -29,6 +30,7 @@ role {
         isa => Dict[
             editor_id => Int,
             text      => Nullable[Str],
+            html      => Nullable[Str],
```

Comment:
We shouldn't need to store the `html` in the database, from what I can see - you're already calling `format_wikitext($self->data->{text})` (so, using `text`) for the display data generation as it's needed.